projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -153,6 +153,46 @@ public function get_share_title( $post_id ) {
 		return html_entity_decode( wp_kses( $title, null ), ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 );
 	}
 
+	/**
+	 * Get a comma-separated list of the post's tags to use for sharing.
+	 * Prepends a '#' to each tag.
+	 *
+	 * @param int $post_id Post ID.
+	 *
+	 * @return string
+	 */
+	public function get_share_tags( $post_id ) {
+		$tags = get_the_tags( $post_id );
+		if ( ! $tags ) {
+			return '';
+		}
+
+		$tags = array_map(
+			function ( $tag ) {
+				// Camel case the tag name and remove spaces as well as apostrophes.
+				$tag = preg_replace( '/\s+|\'/', '', ucwords( $tag->name ) );
+
+				// Return with a '#' prepended.
+				return '#' . $tag;
+			},
+			$tags
+		);
+
+		/**
+		 * Allow customizing how the list of tags is displayed.
+		 *
+		 * @module sharedaddy
+		 * @since $$next-version$$
+		 *
+		 * @param string $tags     Comma-separated list of tags.
+		 * @param int    $post_id  Post ID.
+		 * @param int    $this->id Sharing ID.
+		 */
+		$tag_list = (string) apply_filters( 'jetpack_sharing_tag_list', implode( ', ', $tags ), $post_id, $this->id );
+
+		return html_entity_decode( wp_kses( $tag_list, null ), ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 );
```

Comment:
`wp_kses()` is documented as taking a `string[]|string` for its `$allowed_html` parameter, not `null`.

It looks like, assuming something doesn't blow up on getting passed null, that this will behave the same as the empty string, allowing a small default set of tags.

Reply:
Good point. I had copied from `get_share_title`. I updated both methods in 79ca98343bbd6fc98957c1c8a3c55c1d33fae72f